### ISSUE TITLE
embulk(0.10.21) mkbundle failed with the reason "LoadError: no such file to load -- bundler"

### ISSUE THREAD

**Issue by hiroyuki-sato** [embulk/embulk#1347]
Issue Type: Bug Report
-----------------------

* Write the following environmental information.
  * OS version: macOS 10.15.7
  * Java version: 1.8.0_251
  * Embulk version: 0.10.21
  * Your Embulk configuration (YAML): no config
  * Plugin versions: no plugin

## Write all what you did, e.g. your commands executed

`embulk mkbundle` failed with the reason `LoadError: no such file to load -- bundler`

## Write what you expected and observed actually

```
embulk-dev -l debug mkbundle hoge
2020-11-30 09:29:05.639 +0900 [INFO] (main): embulk_home is set from command-line: /path/to/.embulk-dev
2020-11-30 09:29:05.642 +0900 [DEBUG] (main): Command-line arguments: [hoge]
2020-11-30 09:29:05.642 +0900 [DEBUG] (main): Embulk system properties: {m2_repo=/path/to/.embulk-dev/lib/m2/repository, jruby=file:///path/to/.embulk/jars/jruby-complete-9.1.15.0.jar, gem_path=, gem_home=/path/to/.embulk-dev/lib/gems, embulk_home=/path/to/.embulk-dev, log_level=debug}
Initializing hoge...
  Creating Gemfile
  Creating .bundle/config
  Creating embulk/input/example.rb
  Creating embulk/output/example.rb
  Creating embulk/filter/example.rb
2020-11-30 09:29:08.351 +0900 [INFO] (main): Environment variable "GEM_HOME" is not set. Setting "GEM_HOME" to "/path/to/.embulk-dev/lib/gems" from Embulk system property "gem_home" for the "bundle" command.
LoadError: no such file to load -- bundler
  require at org/jruby/RubyKernel.java:955
  require at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:55
   <main> at <script>:1
org.jruby.embed.EvalFailedException: (LoadError) no such file to load -- bundler
	at org.jruby.embed.internal.EmbedEvalUnitImpl.run(EmbedEvalUnitImpl.java:131)
	at org.jruby.embed.ScriptingContainer.runUnit(ScriptingContainer.java:1307)
	at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1300)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.embulk.jruby.ScriptingContainerDelegateImpl.runScriptlet(ScriptingContainerDelegateImpl.java:802)
	at org.embulk.jruby.LazyScriptingContainerDelegate.runScriptlet(LazyScriptingContainerDelegate.java:303)
	at org.embulk.cli.EmbulkRun.runBundler(EmbulkRun.java:239)
	at org.embulk.cli.EmbulkRun.newBundle(EmbulkRun.java:188)
	at org.embulk.cli.EmbulkRun.runInternal(EmbulkRun.java:78)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:24)
	at org.embulk.cli.Main.main(Main.java:52)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- bundler
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:955)
	at RUBY.require(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:55)
	at RUBY.<main>(<script>:1)
Exception in thread "main" org.jruby.embed.EvalFailedException: (LoadError) no such file to load -- bundler
	at org.jruby.embed.internal.EmbedEvalUnitImpl.run(EmbedEvalUnitImpl.java:131)
	at org.jruby.embed.ScriptingContainer.runUnit(ScriptingContainer.java:1307)
	at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1300)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.embulk.jruby.ScriptingContainerDelegateImpl.runScriptlet(ScriptingContainerDelegateImpl.java:802)
	at org.embulk.jruby.LazyScriptingContainerDelegate.runScriptlet(LazyScriptingContainerDelegate.java:303)
	at org.embulk.cli.EmbulkRun.runBundler(EmbulkRun.java:239)
	at org.embulk.cli.EmbulkRun.newBundle(EmbulkRun.java:188)
	at org.embulk.cli.EmbulkRun.runInternal(EmbulkRun.java:78)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:24)
	at org.embulk.cli.Main.main(Main.java:52)
Caused by: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- bundler
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:955)
	at RUBY.require(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:55)
	at RUBY.<main>(<script>:1)
```

```
alias embulk-dev
alias embulk-dev='embulk-0.10.x -X embulk_home=/path/to/.embulk-dev'
```

```
pwd
/path/to/.embulk-dev
ls -1
embulk.properties
```

```
cat embulk.properties
jruby=file:///path/to/.embulk/jars/jruby-complete-9.1.15.0.jar
```

**Comment by dmikurube** [comment]
The answer would be "Install your own Bundler by `embulk gem install bundler -v 1.16.0`.

**Comment by hiroyuki-sato** [comment]
@dmikurube thank you for your comment.

May I close this issue, or remain open for others?
I think documentation may help for Ruby users.

Should I create another issue?

**Comment by dmikurube** [comment]
I think so. In contrast to #1348 which would be a problem to be solved, this #1347 would never be "fixed" because installing Bundler by users is the intented solution for this issue.

I know documentation would be needed, but we're still on v0.10, the development series. Let us focus on completing the v0.10 development at first.

**Comment by hiroyuki-sato** [comment]
@dmikurube Thank you. Let's close this issue.
